packages/codegen-ui-react/lib/react-component-render-helper.ts

```diff
@@ -631,3 +632,7 @@ export function getSetStateName(stateReference: StateStudioComponentProperty): s
   const stateName = getStateName(stateReference);
   return ['set', stateName.charAt(0).toUpperCase() + stateName.slice(1)].join('');
 }
+
+export function hasChildrenProp(componentProperties: StudioComponentProperties): boolean {
+  return Object.keys(componentProperties).includes('children');
```

Comment:
What about .hasOwnProperty('children') instead of creating and looping through arrays?

Reply:
actually there's a new version https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn

Reply:
much better, thanks.